### PR TITLE
Add timetable menubar

### PR DESCRIPTION
**Pull request by willyb321**
Based on electron and [electron menubar](https://github.com/maxogden/menubar)

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1035,3 +1035,26 @@
     - "research"
     - "spreadsheet"
   icon: "dataproofer.png"
+
```
Comment (kevinsawicki) [inline _data/apps.yml]: Please remove these blank lines, thanks.

**Comment by willyb321** [comment]
Ok, done. Thanks.

File: _data/apps.yml [hunk 2]
```diff
@@ -1035,3 +1035,12 @@
...
   icon: "dataproofer.png"
+-
+  name: "Electron TimeTable"
+  description: "TimeTable in your menubar!"
+  website: "https://github.com/willyb321/electron-menubar-timetable"
+  repository: "https://github.com/willyb321/electron-menubar-timetable"
+  keywords:
+    - "Timetable"
+    - "menubar"
+  icon: "icon.png"
```
Comment (kevinsawicki) [inline _data/apps.yml]: This needs to be the name of the icon you are adding in this pull request, in this case `TTTIcon.png`

File: _data/apps.yml [hunk 3]
```diff
@@ -1035,3 +1035,12 @@
     - "research"
     - "spreadsheet"
   icon: "dataproofer.png"
+-
+  name: "Electron TimeTable"
```
Comment (kevinsawicki) [inline _data/apps.yml]: Mind removing the `Electron` prefix from the name and just call it `TimeTable` instead?

**Comment by kevinsawicki** [comment]
The current icon looks more like an avatar than an app icon, do you have a more application style icon you could use instead?

**Comment by willyb321** [comment]
Alright, all that is done!

**Pull request merged**